client/components/common/CandidateModals/ProvideFeedbackModal/index.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import BasicModal from '../../BasicModal';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faCheck,
+    faCommentAlt,
+    faTimes
+} from '@fortawesome/free-solid-svg-icons';
+import { Form, FormCheck } from 'react-bootstrap';
+import '../common.css';
+import './ProvideFeedbackModal.css';
+
+const ProvideFeedbackModal = ({
+    at = '',
+    handleAction = () => {},
+    issues = [],
+    show = false,
+    testPlan = '',
+    username = ''
+}) => {
+    const [approveInputDisabled, setApproveInputDisabled] = useState('');
+    const [feedbackInputDisabled, setFeedbackInputDisabled] = useState('');
+    const [changesInputDisabled, setChangesInputDisabled] = useState('');
+
+    const radioChange = element => {
+        switch (element.target.id) {
+            case 'approve-input':
+                setApproveInputDisabled('');
+                setFeedbackInputDisabled('disabled');
+                setChangesInputDisabled('disabled');
+                break;
+            case 'feedback-input':
+                setApproveInputDisabled('disabled');
+                setFeedbackInputDisabled('');
+                setChangesInputDisabled('disabled');
+                break;
+            case 'changes-input':
+                setApproveInputDisabled('disabled');
+                setFeedbackInputDisabled('disabled');
+                setChangesInputDisabled('');
+                break;
+        }
+    };
+
+    return (
+        <BasicModal
+            show={show}
+            actionLabel={'Submit Review'}
+            centered={true}
+            content={
+                <div className="feedback-content">
+                    <p>
+                        You have raised{' '}
+                        <a href="#">
+                            {issues.length}{' '}
+                            {issues.length > 1 ? 'issues' : 'issue'}
+                        </a>{' '}
+                        for this test plan.
+                    </p>
+                    <h2 className="feedback-h2">Finish Your Review</h2>
+                    <Form>
+                        <Form.Group>
+                            <FormCheck>
+                                <FormCheck.Input
+                                    name="radio-feedback"
+                                    id="approve-input"
+                                    type="radio"
+                                    onClick={radioChange}
+                                    className={approveInputDisabled}
+                                />
+                                <FormCheck.Label
```

Comment:
I think this label and the others are missing a `for="<id>"` (or `htmlFor` in this case) to point to `<radio>`'s id.

Reply:
Nice catch Howard, that's a good point.